package.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "eslint-config",
```

Should we call it `@pagopa/eslint-config`?
```suggestion
  "name": "@pagopa/eslint-config",
```

---

You are right, is needed the scope into the package name.